src/schema/v2/user.ts

```diff
@@ -44,6 +47,18 @@ export const UserSaleProfileType = new GraphQLObjectType<any, ResolverContext>({
   }),
 })
 
+export const UserAdminNoteType = new GraphQLObjectType<any, ResolverContext>({
+  name: "UserAdminNotes",
+  fields: () => ({
+    ...InternalIDFields,
+    body: {
+      description: "The body of the admin note",
+      type: GraphQLString,
```

Comment:
Is this nullable? (looks like no! https://github.com/artsy/gravity/blob/25e03dddd1071607ec4bd1e181865db313a52bc3/db/schema.rb#L428)